inspirehep/modules/theme/bundles.py

```diff
@@ -63,7 +63,8 @@
         "toastr": "~2.1.2",
         "clipboard": "~1.5.8",
         "flightjs": "~1.5.0",
-        "angular": "~1.4.8",
+        "angular": "~1.4.9",
+        "angular-sanitize": "~1.4.9",
```

Comment:
In principle I don't think we need to add this. `inspirehep-search-js` already depends on that package so it gets installed.